### PR TITLE
Stock metrics webapp

### PR DESCRIPTION
**Pull request by mike2611**
## Changes :partying_face: :computer: 
- Use https://site.financialmodelingprep.com/developer/docs API to obtain information.
- Add routes and navigation into the app.
- Create Home and Details components,
- Use redux to get the info for the components.
- Add styles for Home and Details components.
- Add tests for redux and components.
- Deploy the project.
- Create a descriptive README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,73 @@
...
 
-### `npm start`
+![screenshot](./img/mobileVersion.png)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Live Demo
```
Comment (arslanbisharat) [inline README.md:19]: - [x] Let's make sure you record video of your project as requested in project project guidelines.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

' [ ] let's make sure you follow correct gitflow.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,70 +1,80 @@
...
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Explanation Video
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+<a href="https://www.loom.com/share/c6db8718465647e5a863f4314c313072">
+    <p>Loom | Free Screen & Video Recording Software - Watch Video</p>
+    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/c6db8718465647e5a863f4314c313072-with-play.gif">
+</a>
```
Comment (mikenath223) [inline README.md]: - [x] In the video you are expected to highlight a particular piece of code that you find interesting. Kindly record an additional video discussing this.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**